safehttp/plugins/htmlinject/htmlinject.go

```diff
@@ -106,6 +121,85 @@ func Transform(src io.Reader, cfg ...Config) (string, error) {
 	return rw.out.String(), nil
 }
 
+// LoadTrustedTemplate processes the given TrustedTemplate with the specified default configurations and
+// adds it to the given template.
+// If the given template is nil a new one is created.
+func LoadTrustedTemplate(tpl *template.Template, csp, xsrf bool, src template.TrustedTemplate) (*template.Template, error) {
```

Comment:
These two boolean args next to each other might hurt readability and be error prone.

If you end up opting for a separate package, may I suggest instead:

```
package safehtmlinject

// wrapping so this package controls which ones are safe
type Config struct {
  cfg htmlinject.Config
}

func CSP() Config { return Config{htmlinject.CSPNoncesDefault} }

func XSRF() Config { return XSRF{htmlinject.XSRFTokensDefault } }

func LoadTrustedTemplate(tpl *template.Template, src template.TrustedTemplate, cfg ...Config) (*template.Template, error) { ... }
```

Reply:
Done.